.resinci.yml

```diff
@@ -0,0 +1,41 @@
+---
+npm:
+  environment:
+    - name: CCACHE_TEMPDIR
+      value: /tmp/.ccache-temp
+    - name: CCACHE_COMPRESS
+      value: 1
+    - name: CC
+      value: clang
+    - name: CXX
+      value: clang++
+  platforms:
+    - name: linux
+      os: ubuntu
+      architecture: x86_64
+      node_versions:
+        - "10"
+        - "8"
+        - "6"
+    - name: linux
+      os: ubuntu
+      architecture: x86
+      node_versions:
+        - "8"
+        - "6"
+    - name: darwin
+      os: macos
+      architecture: x86_64
+      node_versions:
+        - "6"
```

Comment:
Should we test Node v8 and v10 on Mac OS & Windows as well?

Reply:
Eventually, yes, but I think we'll need to get things containerized first. Windows in particular will be a nightmare if we don't.